src/node-red/database/influxdb.md

```diff
@@ -1,17 +1,19 @@
 ---
-title: Send data from your Node-RED flows into InfluxDB
-subtitle: A Guide to Connecting InfluxDB with Node-RED
-description: Node-RED has great support for InfluxDB, in this guide we'll explain how to get your data flowing into one of the most popular time-series database.
-date: 2023-07-11
-authors: ["rob-marcer"]
-image: blog/2023/07/images/node-red-influxdb.png
+eleventyNavigation:
+  key: InfluxDB
+  parent: Database
```

Comment:
Should this be plural? What do you think?

Reply:
yes, lets make it plural !